site/public/css/server.css

```diff
@@ -107,9 +107,9 @@ input[type="checkbox"] {
  * TABLE STYLES
  ***************************/
 table {
-    width: 100%;
-    text-align: center;
-    border-collapse: collapse;
+    /*width: 100%;*/
+    /*text-align: center;*/
+    /*border-collapse: collapse;*/
```

Comment:
Fair. I've moved this to a class definition since this change negatively affected students/graders tables.